WordPress/src/main/java/org/wordpress/android/ui/stats/models/InsightsPopularModel.java

```diff
@@ -0,0 +1,46 @@
+package org.wordpress.android.ui.stats.models;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.Serializable;
+
+public class InsightsPopularModel implements Serializable {
+    private int mHighestHour;
+    private int mHighestDayOfWeek;
+    private Double mHighestDayPercent;
+    private Double mHighestHourPercent;
+    private String mBlogID;
+
+    public InsightsPopularModel(String blogID, JSONObject response) throws JSONException {
+        this.setBlogID(blogID);
+        this.mHighestDayOfWeek = response.getInt(("highest_day_of_week"));
+        this.mHighestHour = response.getInt(("highest_hour"));
+        this.mHighestDayPercent = response.getDouble("highest_day_percent");
+        this.mHighestHourPercent = response.getDouble("highest_hour_percent");
+    }
+
+    public String getBlogID() {
+        return mBlogID;
```

Comment:
It's safe. In case of an Exception in the parsing code the default error message is shown in the module UI. I will add default values anyway.